ci-operator/step-registry/rosa/conf/machine-pool/rosa-conf-machine-pool-commands.sh

```diff
@@ -77,6 +77,9 @@ fi
 LOCAL_ZONE_SWITCH=""
 if [[ "$LOCAL_ZONE" == "true" ]]; then
   LOCAL_ZONE_SWITCH=""
+  # Unify rosa localzones macnine pool config with ocp
+  LABELS="node-role.kubernetes.io/edge="
+  TAINTS="node-role.kubernetes.io/edge=:NoSchedule"
```

Comment:
You need to add it like this to not overwrite the customized label/taint:
`LABELS="${LABELS},node-role.kubernetes.io/edge="`

Reply:
Thanks! Addressed.